File: pset6/mario.py
import sys

def main():
    height = int(input("Height: "))
    while height > 23 or height <= 0:
        if height == 0:
            exit(1);
        else:
            height = int(input("Height: "))
    pyramid(int(height))


def pyramid(h):
    for i in range(h):
        for j in range(0, h - i - 1):
            print(" ", end="")

        for k in range(0, i + 1):
            print("#", end="")

        print()


if __name__ == "__main__":
    main()

File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

void pyramid(int h);

int main(void)
{
    printf("Height: ");
    int height = get_int();
    
    
    while (height > 23 || height <=0)
    {
        if (height == 0)
        {
            return 0;
        }
        
        printf("Height: ");
        height = get_int();
    } 
    
    pyramid(height);
    
    
}

void pyramid(int h)
{
    for (int i = 0; i < h; i++)
    {
        for (int j = h - i - 1; j > 0; j--)
        {
            printf(" ");
        }
        
        for (int k = i; k >= 0; k--)
        {
            printf("#");
        }
        
        printf("#\n");
    }
}